### DISCUSSION TITLE
AUTOMATIC PILOT PROCEDURES

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#5223] [Q&A]
Hello, Would it be possible to have a manual of procedures to use the AutoPilot, (Managed mode, Selected mode, with the correct use of the SPD, ALT, V / S buttons) that would make it possible to check that we are applying correctly the correct procedures, before reporting anomalies ...? Thank you very much for your answer and big thank you for your excellent work...

**Comment by MapleNAV-TCASFAULT** [comment]
On our documentation website: https://docs.flybywiresim.com/ you will find upcoming guides on using the entire aircraft, that includes proper use of the autopilot ranging from basic day-to=day operations, to in-depth tips and quirks of the autopilot.

**Reply by kampinod** [reply]
merci beaucoup, je vais voir si je trouve les réponses à mes questions sur ce site...

**Reply by kampinod** [reply]
Je suis retourné voir ce site "docs.flybywiresim", mais je n'ai pas trouvé les réponses à mes questions...En fait j'avais déjà posé des questions au sujet du Pilote Automatique et du MCDU, il y a plusieurs semaines, on m'a répondu que la finalisation du Pilote Automatique était en cours...Donc je voudrais savoir si maintenant le PA respecte les altitudes de descente prévues sur le F-PLN du MCDU, car à priori ce n'est pas le cas chez moi, je dois effectuer la descente en manuel... Est-ce normal...? C'est pourquoi je demande également que l'on publie une procédure, afin de voir si je les applique bien, avant de vous importuner ... Merci pour votre travail...

**Reply by kampinod** [reply]
Réponse à moi-même : j'utilisais depuis plusieurs semaines la version "Stable" (j'avais abandonné la version DEV, car trop de bugs) je viens donc de réinstallé la version "DEV", et j'ai effectué un vol LFMN-FLPG et j'ai effectué la descente en mode "Managed" et tout s'est déroulé normalement, je suis arrivé au point Initial Fix à 5000 fts comme prévu... donc plus de problème, mais à priori la version "Stable" ne permet pas d'effectuer correctement une descente en mode "Managed" sauf erreur de ma part. En tout cas grand merci pour votre travail...
